### PR TITLE
Add ember-assign-polyfill

### PR DESCRIPTION
**Pull request by rwwagner90**
@mixonic this is just adding ember-assign-polyfill on top of #427

### CODE REVIEW

**Comment by rwwagner90** [comment]
@mixonic any chance we could merge this?

**Comment by mixonic** [comment]
What is the intent of this change? @rwwagner90 Improve compatibility with Ember 3.0, or to avoid a deprecation warning on a certain version?

It looks like CI is gone/missing. Did you confirm this passes the whole ember try suite?

**Comment by rwwagner90** [comment]
@mixonic all of the above. Lots of deprecations thrown from using merge. We had several thousand deprecation warnings in an app I was working on, coming from torii.

Not sure what happened to CI. I just ran ember-try locally. Everything is good to go, just had to disable a lint rule.

Screen Shot 2019-08-17 at 9 49 02 AM

File: .eslintrc.js [hunk 1]
```diff
@@ -15,7 +15,8 @@ module.exports = {
     browser: true
   },
   rules: {
-    "ember/avoid-leaking-state-in-ember-objects": 0
+    'ember/avoid-leaking-state-in-ember-objects': 'off',
+    'ember/no-side-effects': 'off'
```
Comment (mixonic) [inline .eslintrc.js]: What required this?

Comment (rwwagner90) [reply .eslintrc.js]: @mixonic it's been broken. All the CI runs have been failing because of a side effect somewhere.

**Comment by rwwagner90** [comment]
@mixonic any chance we could merge this soon? Happy to explain anything further, if need be.

**Comment by Dhaulagiri** [comment]
@rwwagner90 can you rebase this against master now that the test suite should be working again

**Comment by Dhaulagiri** [comment]
It's still worth a shot, but it looks like the tests are not actually running on master due to issues with circle ci I'm not sure I have access to fix

**Comment by rwwagner90** [comment]
@Dhaulagiri are you saying we should remove the addition of `'ember/no-side-effects': 'off'`?

**Comment by Dhaulagiri** [comment]
No, I think the problem is unrelated to this PR

**Comment by rwwagner90** [comment]
@Dhaulagiri yeah, the problem is unrelated, but you had asked if I could rebase, but there are no conflicts here, so I am looking for what you would like me to do? The only change that maybe should be removed is `'ember/no-side-effects': 'off'` but otherwise this is good to go.

**Comment by Dhaulagiri** [comment]
My thought was it’d get the tests to run with the updated testing setup I just merged into master. But then I realized the tests aren’t actually running...yet

**Comment by rwwagner90** [comment]
@Dhaulagiri so can we go ahead and merge? The tests were passing before.

**Comment by rwwagner90** [comment]
@mixonic @Dhaulagiri is there anything blocking us from merging this? It should be zero risk, since it's only changing merge to assign and also adding ember-assign-polyfill to ensure it continues to work if Ember.assign is not present.

**Comment by damatri** [comment]
Yes, any chances this will be merged soon? This is giving deprecation warnings since 3.6.0.

**Comment by rwwagner90** [comment]
We may need to fork Torii and release a new package.

**Comment by damatri** [comment]
Guess this package is unmaintained with 70 issues and 30 pull requests..

**Comment by rwwagner90** [comment]
@mixonic @Dhaulagiri should we fork?

**Comment by rwwagner90** [comment]
That would be great!

**Comment by rwwagner90** [comment]
@Dhaulagiri any updates on moving to adopted-ember-addons?

**Comment by rwwagner90** [comment]
@Dhaulagiri @mixonic if there is no desire to merge this, I will close soon. It has been open a long time. I will also be forking this project in the future, if I have a need to use it again.

**Comment by rwwagner90** [comment]
Since we haven't been able to merge this in a year, I am going to close.

**Pull request closed without merging**